docs-2.0/7.data-security/1.authentication/4.ldap.md

```diff
@@ -1,39 +1,38 @@
-<!--
-# 使用OpenLDAP进行身份验证
+# OpenLDAP authentication
 
-本文介绍如何将Nebula Graph连接到OpenLDAP服务器，使用OpenLDAP中定义的DN（Distinguished Name）和密码进行身份验证。
+This topic introduces how to connect Nebula Graph to the OpenLDAP server and use the DN (Distinguished Name) and password defined in OpenLDAP for authentication.
 
 !!! enterpriseonly
 
-    仅企业版支持本功能。
+    This feature is supported by the enterprise edition only.
 
-## 认证方式
+## Authentication method
 
-启用OpenLDAP身份验证后，输入用户的账号和密码登录Nebula Graph时，Nebula Graph会在Meta服务中查找登录账号是否存在，如果账号存在，再根据认证方式去OpenLDAP中找到对应的DN，验证密码。
+After the OpenLDAP authentication is enabled and users log in to Nebula Graph with the account and password, Nebula Graph checks whether the login account exists in the Meta service. If the account exists, Nebula Graph finds the corresponding DN in OpenLDAP according to the authentication method and verifies the password.
 
-OpenLDAP支持的认证方式有两种：简单绑定认证和搜索绑定认证。
+OpenLDAP supports two authentication methods: simple bind authentication (SimpleBindAuth) and search bind authentication (SearchBindAuth).
 
-### 简单绑定认证（SimpleBindAuth）
+### SimpleBindAuth
 
-简单绑定认证会根据登录账号和Graph服务配置信息，拼接成OpenLDAP可以识别的DN，然后根据DN和密码，在OpenLDAP上进行验证。
+Simple bind authentication splices the login account and the configuration information of Graph services into a DN that can be recognized by OpenLDAP, and then authenticates on OpenLDAP based on the DN and password.
 
-### 搜索绑定认证（SearchBindAuth）
+### SearchBindAuth
 
-搜索绑定认证会读取Graph服务配置信息，查询配置信息中的`uid`和登录账号是否匹配，如果匹配，就读取这个DN，然后用DN和密码，在OpenLDAP上进行验证。
+Search bind authentication reads the Graph service configuration information and queries whether the `uid` in the configuration matches the login account. If they match, search bind authentication reads the DN, and then uses the DN and password to verify on OpenLDAP.
 
-## 前提条件
+## Prerequisites
 
-- 已安装[OpenLDAP](https://www.openldap.org/)。
+- [OpenLDAP](https://www.openldap.org/) is installed.
 
-- 已在OpenLDAP上导入用户的账号和密码信息。
+- The account and password are imported on OpenLDAP.
 
-- OpenLDAP所在服务器已开放相应认证端口。
+- The server where OpenLDAP is located opens the corresponding authentication port.
```

Comment:
`has opened` or `the ports are open`
`opens` does not indicate that this operation has been done.